graphql-api/src/main/resources/stream-registry.graphql

```diff
@@ -1,4 +1,5 @@
 scalar ObjectNode
+scalar Long
```

Comment:
Looking at the apollo client code it does support longs - can you double check this?

Reply:
graphql-java does but graphql doesn't. We need to be super careful here, particularly if we will have js clients. 

https://graphql.org/graphql-js/basic-types/

Reply:
Presumably we should be using the GraphQLLong scalar type?

Reply:
I don't have an anwser to this :)

Reply:
I think we should prefer an existing type rather than defining our own scalar where possible.

Reply:
But does it exist? Long is not part of graphql spec.

Reply:
No, but it is part of the graphql-java source, so in that sense it is 'canon' and for example any js client is (more) likely to be able to handle it.